dtool/LocalSetup.cmake

```diff
@@ -193,7 +193,14 @@ else()
   set(intdir "${CMAKE_BUILD_TYPE}")
 endif()
 
-configure_file(dtool_config.h.in "${PROJECT_BINARY_DIR}/${intdir}/include/dtool_config.h")
+configure_file(dtool_config.h.in "${PROJECT_BINARY_DIR}/include/dtool_config.h")
+
+# TODO: Add the ability to customize dtool_config.h based on configuration.
+if(IS_MULTICONFIG)
```

Comment:
We need to talk more about what behavior we want when generating dtool_config.h for multi-config generators. For example, do we want to use generator expressions or have a different .in file for each config?

Reply:
We'll want to iterate over all potential configs and generate the dtool_config.h for each. This also implies a mechanism for detecting when options that are config-specific are left at their defaults, so the per-config default can be used.

Reply:
Would it be easier to generate a single `dtool_config.h` that sets the values for different configs using some ifdefs?